bindings/wasm/src/did/wasm_document.rs

```diff
@@ -107,6 +192,20 @@ impl WasmDocument {
   // Verification Methods
   // ===========================================================================
 
+  /// Returns a list of all {@link VerificationMethod} in the DID Document.
+  #[wasm_bindgen(js_name = methods)]
+  pub fn methods(&self) -> ArrayMethods {
+    self
+      .0
+      .methods()
+      .collect::<Vec<&IotaVerificationMethod>>()
+      .into_iter()
+      .cloned()
```

Comment:
Nit: unnecessary `collect`.

```suggestion
      .methods()
      .cloned()
```